content/markdown/concepts/links/links.md

```diff
@@ -31,32 +31,46 @@ We just updated our [Docs Contribution Guide](https://www.codecademy.com/pages/c
 
 ## Accessibility: Link Titles
 
-A title for a link can be optionally added. This will appear as a tooltip when the user hovers over the link. To add a title, enclose it in parentheses `()` after the URL.
+A title for a link can be optionally added. This will appear as a tooltip when the user hovers over the link. To add a title, enclose it in quotation marks `""` after the URL inside the parentheses.
 
-Links should have link titles so that screenreaders can narrate them as well as a user can <kbd>Tab</kbd> through the page.
+Including link titles improves accessibility by providing additional context for screen readers and users navigating with the keyboard.
 
 ```md
-[Link text](https://website-name.com 'Link title')
+[Link text](https://website-name.com "Link title")
 ```
 
 For example:
 
 ```md
-My favorite Craigslist category is [Missed Connections](https://newyork.craigslist.org/d/missed-connections/search/mis 'The best place on the internet').
+My favorite Craigslist category is [Missed Connections](https://newyork.craigslist.org/d/missed-connections/search/mis "The best place on the internet").
 ```
 
 The output would look like:
 
-My favorite Craigslist category is [Missed Connections](https://newyork.craigslist.org/d/missed-connections/search/mis 'The best place on the internet').
+My favorite Craigslist category is [Missed Connections](https://newyork.craigslist.org/d/missed-connections/search/mis "The best place on the internet").
 
 ## URLs and Email Addresses
 
-To quickly turn a URL or email address into a link, enclose it in angle brackets `<` `>`.
+To turn a URL or email address into a clickable link, enclose it in angle brackets `<URL or email>`.
 
 - `<https://www.codecademy.com/resources/docs>`
-- `<hotmale@hotmail.com>`
+- `<demoemail@hotmail.com>`
 
 The rendered output looks like this:
 
 - <https://www.codecademy.com/resources/docs>
-- <hotmale@hotmail.com>
+- <demoemail@hotmail.com>
+
+## Internal Linking
+
+To create links to navigate to other sections within the same document, or sections within other documents on the same website, use a relative path (when linking to sections in other documents) with the `#` symbol, followed by the section's title. Ensure the title is in lowercase, omit any punctuation and replace spaces with hyphens.
```

Comment:
can you make this sentence short? or just divide it into multiple sentences.

Reply:
Hi, I'm new to open source so apologies in case I've missed anything!

I've split the sentence into two so it reads like this. Could also remove the bit in brackets, I think it's redundant but another reviewer added this earlier so I thought I should just leave it in for now.

```
To create links to navigate to other sections within the same document, or sections within other documents on the same website, use a relative path (when linking to sections in other documents) with the `#` symbol, followed by the section's title. Ensure the title is in lowercase, omit any punctuation and replace spaces with hyphens.
```